setup.py

```diff
@@ -8,7 +8,7 @@
 except IOError:
     README = ""
 
-trigger_extras = {"PyYAML<6", "lxml>=4.3.1", "mock==3.*"}
+trigger_extras = {"PyYAML==6.0", "lxml>=4.3.1", "mock==3.*"}
```

Comment:
could you use `<7` instead? We can let the app decide on a more specific version